modules/exploits/linux/http/fusionpbx_php_editor_cmd_exec.rb

```diff
@@ -0,0 +1,181 @@
+####################################################################
```

Comment:
Please use the standard header.

Reply:
This isn't the standard header?

Reply:
Correct. It is not the standard header. This is the standard header:

```ruby
##
# This module requires Metasploit: https://metasploit.com/download
# Current source: https://github.com/rapid7/metasploit-framework
##
```